src/url_pattern_helpers.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ada/url_pattern_helpers.h"
+
+namespace ada::url_pattern_helpers {
+
+std::vector<int> tokenize() {
+  std::vector<int> token_list;
+  return token_list;
```

Comment:
what happens if you return std::move(token_list) here? @jasnell's recommendation